src/Bicep.LangServer/Handlers/BicepCodeActionHandler.cs

```diff
@@ -67,12 +74,88 @@ public override Task<CommandOrCodeActionContainer> Handle(CodeActionParams reque
 
             commandOrCodeActions.AddRange(analyzerDiagnostics);
 
+            var diagnosticsOfTypeWarningAndInfo = diagnostics
+                .Where(diagnostic =>
+                    (diagnostic.Level == DiagnosticLevel.Warning || diagnostic.Level == DiagnosticLevel.Info) &&
+                    (diagnostic.Span.ContainsInclusive(requestStartOffset) ||
+                    diagnostic.Span.ContainsInclusive(requestEndOffset) ||
+                    (requestStartOffset <= diagnostic.Span.Position && diagnostic.GetEndPosition() <= requestEndOffset)))
+                .OfType<IDiagnostic>();
```

Comment:
Aren't they already of this type?

Reply:
Ahh yes. Thanks!

Updated as part of this commit:
https://github.com/Azure/bicep/commit/7f012613ae1aad9fd2e8101ac82b73b362b8686b